molecule/default/converge.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
+    - make_cmd: 'make -s install'
+
+  pre_tasks:
+
+    - name: Update apt cache.
+      apt: update_cache=true cache_valid_time=600
+      changed_when: false
+
+    - name: "Installing Rsync"
+      apt: >
```

Comment:
Same as all the rest of the items that now test centos - apt is ubuntu/debian-specific.

Reply:
Fixed those.

This runs ok in local. But throws the "rsync" error in github.